ios/UnflowMetadataBridge.swift

```diff
@@ -9,6 +9,8 @@ import Foundation
 import UnflowUI
 
 struct UnflowMetadataBridge {
+    static let dateFormatter = ISO8601DateFormatter()
```

Comment:
Just tested and this doesn't parse the dates correctly. I think we need to add these two fomatting options
```
dateFormatter.formatOptions = [.withInternetDateTime, .withFractionalSeconds]
```

I added them just above the `date` method but wasn't sure if there is a way to do it statically